src/jdk.jdwp.agent/share/native/libjdwp/utf_util.c

```diff
@@ -343,6 +343,7 @@ static WCHAR* getWideString(UINT codePage, char* str, int len, int *pwlen) {
     }
     if (MultiByteToWideChar(codePage, 0, str, len, wstr, wlen) == 0) {
         UTF_ERROR(("Can't get WIDE string"));
+        free(wstr);
```

Comment:
Is this really an issue? I thought that UTF_ERROR prints the error and aborts the program.

Reply:
Hi Alan, yeah I saw  that it prints an error but you are correct it does an abort() call too. So probably we can keep this as it is.
Another  MultiByteToWideChar / malloc  case I just saw 
https://github.com/openjdk/jdk/blob/master/src/java.instrument/windows/native/libinstrument/EncodingSupport_md.c#L79
shouldn't there be the free call after line 80 ?

Reply:
> Another MultiByteToWideChar / malloc case I just saw
> https://github.com/openjdk/jdk/blob/master/src/java.instrument/windows/native/libinstrument/EncodingSupport_md.c#L79
> shouldn't there be the free call after line 80 ?

I think you are right, it won't free if MultiByteToWideChar were to fail.